src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -161,7 +160,29 @@ def register_submission(
                 get_drc=get_drc,
                 language=language_code_2b,
             )
-            record_data["csv_url"] = submission_csv_document["url"]
+            csv_url = submission_csv_document["url"]
+
+        base_render_context = {
+            "_submission": submission,
```

Comment:
After discussing with DH this morning, it is important that the public reference is included here. For the confirmation email we have in the context:
```
...
"public_reference": submission.public_registration_reference
...
```
so it would be good to add that here too.